src/controllers/employeeDetails.controller.js

```diff
@@ -24,6 +24,39 @@ const {employeeValidation,getId,getListValidation,recoveryParamsValidation,updat
     }
   };
 
+  //create Employee details with condition
+  const addOne = async (req ,res ,next) => {
+    const reqData = { ...req.query };
+    try {
+    const { employee_id, 
+      department_id, 
+      first_name, 
+      last_name, 
+      address,
+      email_id, 
+      phone_no,}  = await employeeValidation.validateAsync(req.body);
+    const { page_size,page_no } = await getListValidation.validateAsync(reqData);
+    const department = await departmentService.getAll({ page_size, page_no });
+    console.log('-------->>>>',department);
```

Comment:
console.log is stritly prohibited

Reply:
PLease try to avoid console.log
If it the requirement than you can go with console.info or console.info